dashboards.go

```diff
@@ -60,516 +42,21 @@ var commandDashboards = cli.Command{
 				cli.StringFlag{Name: "file-path, F", Usage: "read dashboard from the file"},
 			},
 		},
+
+		// urfave/cli will call default Action when running undefined subcommands,
+		// So we leave command defintion to cause error when executing `mkr dashboards generate`.
 		{
-			Name:      "migrate",
```

Comment:
I decided to remove migrate command from command definition.  because

* this command was only used for a short period of time
* it was not for an automation tools